backend/src/interceptors/remove-users-fields/remove-users-fields.interceptor.ts

```diff
@@ -18,7 +18,8 @@ export class RemoveUsersFieldsInterceptor implements NestInterceptor {
         }
         delete user.refreshToken;
         delete user.mfaSecret;
-        delete user.mfaEnabled;
+        // NOTE Is it necessary to remove mfaEnabled?
```

Comment:
Is it necessary to delete mfaEnabled from user requests?
If this is a security issue, we will need to change part of the 2FA architecture because this is currently public in the payload part of the JWT.

Reply:
Well, I think that if we just send whether it is enabled or not, it's not a problem. Only the secret is dangerous here.